src/fastmcp/cli/install/stdio.py

```
@@ -0,0 +1,156 @@
+
+        # Build the full command
+        full_command = env_config.build_command(["fastmcp", "run", server_spec])
+        command_str = shlex.join(full_command)
+
+        if copy:
+            pyperclip.copy(command_str)
+            rich_print("[green]✓ Command copied to clipboard[/green]")
+        else:
+            builtins.print(command_str)
+
+        return True
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)